helm/values.yaml

```
@@ -17,6 +17,9 @@ coder:
     # coder.image.pullPolicy -- The pull policy to use for the image. See:
     # https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
     pullPolicy: IfNotPresent
+    # coder.image.pullSecret -- The secret used for pulling the Coder image from
+    # a private registry.
+    pullSecret: ""
```

To avoid having to change this in the future we should make this an array and pass it through with `toYaml` like other values.

```suggestion
    pullSecret: []
      # - name: "pull-secret"
```